ciao-cli/image.go

```diff
@@ -97,18 +87,11 @@ var imageCommand = &command{
 }
 
 type imageAddCommand struct {
-	Flag            flag.FlagSet
-	name            string
-	id              string
-	containerFormat imageContainerFormat
-	diskFormat      imageDiskFormat
-	minDiskSize     int
```

Comment:
And it's not image size, it's the required disk for the OS to run. A VM with only 100M of available disk would not be really useful in real worlds scenarios.